conbench/templates/index.html

```diff
@@ -63,7 +63,7 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
                      <td><a href="{{ run.commit.repository }}">{{ run.commit.display_repository }}</a></td>
 
                      {% if run.commit.display_message %}
-                       <td><a href="{{ run.commit.url }}">{{ run.commit.display_message }}</a></td>
+                       <td><a href="{{ run.commit.url }}">{{ run.commit.display_message }} {{ run.commit.sha[:7] }}</a></td>
```

Comment:
One suggestion: we could put this hash up front instead of after the commit message. That way we could truncate the commit message but still allow folks to get to that data when they click through to the commit.